backend/src/main/java/ca/bc/gov/app/exception/UnexpectedErrorException.java

```diff
@@ -6,7 +6,11 @@
 
 @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
 public class UnexpectedErrorException extends ResponseStatusException {
+
+  private static final long serialVersionUID = 6298011852284611032L;
```

Comment:
You don't need to add the serial version anymore. This is only used for binary serialization and serves no purpose when handling JSON data

Reply:
Thanks. My IDE was complaining for that, so I added it :)